core-bundle/src/Controller/AbstractController.php

```diff
@@ -140,4 +145,12 @@ protected function hasParameter(string $name): bool
 
         return $this->container->get('parameter_bag')->has($name);
     }
+
+    protected function createBackendResponseContext(): ResponseContext
```

Comment:
The return value is not used in the `BackendController` class. Is this intended?

Reply:
Yes - in your own controller you might want to create the response context and then immediately fill its `HtmlHeadBag` with values. The `BackendController::mainAction` only needs to create it (for the `BackendMain` legacy controller) without needing to access it immediately.